data/tools/converters/HtmlToJsx.ts

```diff
@@ -0,0 +1,12 @@
+import { Tool } from "@/data/types";
+
+const HtmlToJsx: Tool = {
+  id: 2,
+  name: "Html To Jsx",
+  description: "Convers Html To Jsx and vice versa",
+  componentFileName: "HtmlToJsx",
+  slug: "html-to-jsx",
+  categorySlug: "converters",
```

Comment:
I was actually working on a new feature. That requires you to add an `acceptedDataTypes` field which is an array of accepted data types. I have made it optional now but I guess you still have to add it. Just add the following - 

```ts
const HtmlToJsx: Tool = {
  id: 2,
  name: "Html To Jsx",
  description: "Convers Html To Jsx and vice versa",
  componentFileName: "HtmlToJsx",
  slug: "html-to-jsx",
  categorySlug: "converters",
  acceptedDataTypes: ["html", "jsx"],  // add this line
}

Reply:
Ok